src/main/java/com/foo/unmls/UnmlsGraphTool.kt

```diff
@@ -0,0 +1,41 @@
+package com.foo.unmls
+
+import com.github.ajalt.clikt.core.CliktCommand
+import com.github.ajalt.clikt.core.subcommands
+import com.github.ajalt.clikt.parameters.options.option
+import com.github.ajalt.clikt.parameters.options.required
+import com.github.ajalt.clikt.parameters.types.int
+
+object UnmlsGraphTool {
+    @JvmStatic
+    fun main(args: Array<String>) {
+        Cli().main(args)
+    }
+}
+
+private class Cli : CliktCommand() {
+    init {
+        subcommands(
+            Foo(),
+            Bar()
+        )
+    }
+
+    override fun run() {
+        // no op
+    }
+}
+
+private class Foo : CliktCommand() {
+    private val widgetSize: Int by option().int().required()
```

Comment:
this will produce a `--widget-size` CLI param automatically (can customize the name, but by default it goes off of the property name)